src/bot.js

```diff
@@ -279,35 +262,28 @@ class HangoutsChatBot extends Adapter {
         this.robot.logger.error('Unrecognized event type: ' + event.type);
         return;
     }
-
-    // Pass the message to the Hubot bot
-    this.robot.receive(hangoutsChatMessage, () => {
-      // For HTTP bots, only `res.reply()` uses the synchronous response. Hence,
-      // if a bot handles the message without calling reply, an HTTP response
-      // won't be sent. In this case, Hangouts Chat will assume that the request
-      // timed out and retry. To avoid this retry behavior, we automatically
-      // send a success response if the user called `res.send()` or if it is a
-      // REMOVED_FROM_SPACE event. The user can also force this behavior by
-      // calling message.setHandled() or message.finish().
-      if (!this.isPubSub
-          && (hangoutsChatMessage.handled || hangoutsChatMessage.done)
-          && !res.headersSent) {
-        res.sendStatus(200);
-      }
+    this.robot.receive(hangoutsChatMessage, ()=>{
+      hangoutsChatMessage.finish();
+      hangoutsChatMessage.setHandled();
```

Comment:
It looks like you'd like to change the logic to always respond to the request, which I agree is a good idea. In this case, we don't need to mark the message as finished / handled.
Please remove references to finish/setHandled. You can leave the `setHandled` function definition to not break any existing users.

Reply:
Yes, the intent is to ensure they get called at the right time.

Reply:
SG, please remove `finish`/`setHandled` if they're not needed.